source/fuzz/repeated_pass_instances.h

```diff
@@ -0,0 +1,173 @@
+  REPEATED_PASS_INSTANCE(AddBitInstructionSynonyms);
+  REPEATED_PASS_INSTANCE(AddCompositeInserts);
+  REPEATED_PASS_INSTANCE(AddCompositeTypes);
+  REPEATED_PASS_INSTANCE(AddCopyMemory);
+  REPEATED_PASS_INSTANCE(AddDeadBlocks);
+  REPEATED_PASS_INSTANCE(AddDeadBreaks);
+  REPEATED_PASS_INSTANCE(AddDeadContinues);
+  REPEATED_PASS_INSTANCE(AddEquationInstructions);
+  REPEATED_PASS_INSTANCE(AddFunctionCalls);
+  REPEATED_PASS_INSTANCE(AddGlobalVariables);
+  REPEATED_PASS_INSTANCE(AddImageSampleUnusedComponents);
+  REPEATED_PASS_INSTANCE(AddLoads);
+  REPEATED_PASS_INSTANCE(AddLocalVariables);
+  REPEATED_PASS_INSTANCE(AddLoopPreheaders);
+  REPEATED_PASS_INSTANCE(AddOpPhiSynonyms);
+  REPEATED_PASS_INSTANCE(AddParameters);
+  REPEATED_PASS_INSTANCE(AddRelaxedDecorations);
+  REPEATED_PASS_INSTANCE(AddStores);
+  REPEATED_PASS_INSTANCE(AddSynonyms);
+  REPEATED_PASS_INSTANCE(AddVectorShuffleInstructions);
+  REPEATED_PASS_INSTANCE(ApplyIdSynonyms);
+  REPEATED_PASS_INSTANCE(ConstructComposites);
+  REPEATED_PASS_INSTANCE(CopyObjects);
+  REPEATED_PASS_INSTANCE(DonateModules);
+  REPEATED_PASS_INSTANCE(DuplicateRegionsWithSelections);
+  REPEATED_PASS_INSTANCE(FlattenConditionalBranches);
+  REPEATED_PASS_INSTANCE(InlineFunctions);
+  REPEATED_PASS_INSTANCE(InvertComparisonOperators);
+  REPEATED_PASS_INSTANCE(MakeVectorOperationsDynamic);
+  REPEATED_PASS_INSTANCE(MergeBlocks);
+  REPEATED_PASS_INSTANCE(MutatePointers);
+  REPEATED_PASS_INSTANCE(ObfuscateConstants);
+  REPEATED_PASS_INSTANCE(OutlineFunctions);
+  REPEATED_PASS_INSTANCE(PermuteBlocks);
+  REPEATED_PASS_INSTANCE(PermuteFunctionParameters);
+  REPEATED_PASS_INSTANCE(PermuteInstructions);
+  REPEATED_PASS_INSTANCE(PropagateInstructionsUp);
+  REPEATED_PASS_INSTANCE(PushIdsThroughVariables);
+  REPEATED_PASS_INSTANCE(ReplaceAddsSubsMulsWithCarryingExtended);
+  REPEATED_PASS_INSTANCE(ReplaceCopyMemoriesWithLoadsStores);
+  REPEATED_PASS_INSTANCE(ReplaceCopyObjectsWithStoresLoads);
+  REPEATED_PASS_INSTANCE(ReplaceLoadsStoresWithCopyMemories);
+  REPEATED_PASS_INSTANCE(ReplaceIrrelevantIds);
+  REPEATED_PASS_INSTANCE(ReplaceOpPhiIdsFromDeadPredecessors);
+  REPEATED_PASS_INSTANCE(ReplaceOpSelectsWithConditionalBranches);
+  REPEATED_PASS_INSTANCE(ReplaceParameterWithGlobal);
+  REPEATED_PASS_INSTANCE(ReplaceLinearAlgebraInstructions);
+  REPEATED_PASS_INSTANCE(ReplaceParamsWithStruct);
+  REPEATED_PASS_INSTANCE(SplitBlocks);
+  REPEATED_PASS_INSTANCE(SwapBranchConditionalOperands);
+#undef REPEATED_PASS_INSTANCE
+
+ public:
+  // Yields the sequence of fuzzer pass instances that have been registered.
+  const std::vector<std::unique_ptr<FuzzerPass>>& GetPasses() const {
+    return passes_;
+  }
+
+ private:
+  // The distinct fuzzer pass instances that have been registered via SetPass().
+  std::vector<std::unique_ptr<FuzzerPass>> passes_;
+};
+
+}  // namespace fuzz
+}  // namespace spvtools
+
+#endif  // SOURCE_FUZZ_REPEATED_REPEATED_PASS_INSTANCES_
```

Comment:
`SOURCE_FUZZ_REPEATED_REPEATED_PASS_INSTANCES_H_`.

Reply:
The same for other haders.

Reply:
Haha, something funny about "REPEATED_REPEATED" :)  Thanks for spotting this.

Reply:
A metajoke :)